spec/charts/bar_chart/bar_chart_spec.rb

```diff
@@ -255,10 +255,10 @@
       labels_texts = svg.css('text.label').map{ |t| t.text.tr("\n", '') }
       expect(labels_texts).to eq(labels)
     end
-    describe 'too few labels' do
+    describe 'count of label and data does not match' do
       let(:labels) { ['Alpha', 'Beta'] }
       it 'raises an error' do
-        expect{ chart.render }.to raise_error(ArgumentError)
+        expect{ chart.render }.to raise_error(ArgumentError, 'number of labels does not match array')
```

Comment:
Unless you are testing for the actual error message, there doesn't need to be one.  There could be a message that indicates the details of the test upon failure though... (I think, would have to check the test framework documentation).